pkg/cluster/root_app.go

```diff
@@ -60,8 +63,15 @@ func ConstructRootApp(
 		app.ObjectMeta.Labels["arlon-cluster"] = clusterName
 		// assume CAPI for now
 		apiProvider = "capi"
+		if withCAS {
+			err := enableClusterAutoscaler(apiProvider, &helmParams, &ignoreDiffs)
+			if err != nil {
+				return nil, err
+			}
+			fmt.Println(helmParams, ignoreDiffs)
```

Comment:
Library functions must not dump to stdout.

Reply:
Just for debugging will remove before merge

Reply:
done